src/Room.ts

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose';
```

Comment:
use typegoose, I've already created the schema. https://github.com/yesh0907/Toohak-BE/pull/3